vernac/vernacexpr.ml

```diff
@@ -44,6 +44,7 @@ type printable =
   | PrintGraph
   | PrintClasses
   | PrintTypeClasses
+  | PrintTypeclasses
```

Comment:
Is it really necessary to add this?

Reply:
Yes because the code is incorrectly capitalized too. When we finish the deprecation we will get rid of the old.

Reply:
The constructor `PrintTypeClasses` might read as a command `Print Type Classes` in the readers head. I am correcting it to be `PrintTypeclasses`.